### PR TITLE
Make rEFInd installation simpler + improvements to startup manager guide

### PR DESCRIPTION
**Pull request by AdityaGarg8**

### CODE REVIEW

**Comment by AdityaGarg8** [comment]
Works for me

**Comment by AdityaGarg8** [comment]
cc @Redecorating

**Review by Redecorating — CHANGES_REQUESTED** [review]

File: docs/guides/refind.md [hunk 1]
```diff
@@ -12,49 +12,25 @@ All steps given here have to be performed on **macOS**. You will also need to ha
...
-    6:                APFS Volume ⁨VM⁩                      20.5 KB    disk1s5
-    ```
-  
-    Here, the disk indentifier of `REFIND` volume is `disk0s4`.
-
-5. Now run the following in the terminal. Make sure you replace `disk0s4` (found in 4th, 5th, 6th and 7th line of the command given below) with the disk identifier you got in the output as described in step 4.
+4. Now run the following in the terminal :-
 
     ```plain
+    IDENTIFIER=$(diskutil info REFIND | grep "Device Identifier" | cut -d: -f2 | xargs)
```
[the remark this answers is not in the document]
Comment (AdityaGarg8) [reply docs/guides/refind.md:18]: Without `xargs` there are blank spaces in the output.

[the remark this answers is not in the document]
Comment (Redecorating) [reply docs/guides/refind.md:18]: Ah that makes sense.

File: docs/guides/startup-manager.md [hunk 2]
```diff
@@ -10,11 +10,17 @@ The steps to perform the above tasks vary as per the way you have installed Linu
 
 In this case, boot into macOS, open a terminal window and run :-
 
-`bless --folder /Volumes/EFI/EFI/BOOT --label <YOUR DISTRO'S NAME>`
+```plain
```
[the remark this answers is not in the document]
Comment (AdityaGarg8) [reply docs/guides/startup-manager.md]: check now

**Review by Redecorating — APPROVED** [review]

**Pull request merged**